docker-storage-setup.sh

```diff
@@ -978,7 +976,7 @@ setup_extra_volume() {
 }
 
 setup_extra_lv_fs() {
-  [ -z "$CONTAINER_ROOT_LV_MOUNT_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
+  [ -z "$RESOLVED_MOUNT_DIR_PATH" ] && [ -z "$CONTAINER_ROOT_LV_NAME" ] && return 0
```

Comment:
Ideally this change should be in a separate patch as it has nothing to do with renaming of variables. Even if you leave it here, I am fine.